packages/rum-core/src/performance-monitoring/metrics.js

```diff
@@ -332,7 +329,7 @@ export class PerfEntryRecorder {
        *   buffered flag (https://w3c.github.io/performance-timeline/#observe-method)
        */
       this.po.observe({ type, buffered: true })
-    } catch (_) {}
+    }
```

Comment:
We added try/catch to avoid errors in unsupported browsers. Unsupported browsers like safari 10 would throw error. It's safer to have this.